src/pages/fi-accounts/forms/CreditLimitsForm.js

```
@@ -1,7 +1,137 @@
+            limit: ''
+          }
+        ]
+      },
+      onSubmit: values => {
+        postCurrencies(values.currencies)
+      }
+    })
+
+    const postCurrencies = obj => {
+
+      const filteredObj = obj.filter(({ limit }) => limit > 0);
+
+      const saveCurrency = filteredObj.map(currency => {
+        const data = {
+          accountId: currency.accountId,
+          currencyName: currency.currencyName,
+          currencyId: currency.currencyId,
+          limit: currency.limit
+        };
+    
+        return postRequest({
+          extension: FinancialRepository.AccountCreditLimit.set,
+          record: JSON.stringify(data)
+        })
+      });
+      Promise.all(saveCurrency)
+      .then(res => {
+        if (res) toast.success('Record Edited Successfully')
+        getCurrencies(accountId)
+      })
+      
+      .catch(error => {
+      })
+    }
+
+    const column = [
+      {
+        component: 'textfield',
+        label: labels.currency,
+        name: 'currencyName',
+        props:{readOnly: true}
+      },
+      {
+        component: 'textfield',
+        label: labels.CreditLimits,
+        name: 'limit',
+        props:{type:'number'}
+      }
+    ]
+
+    useEffect(()=>{
+      accountId  && getCurrencies(accountId)
+    }, [accountId])
+
+    const getCurrencies = accountId => {
+      const defaultParams = `_accountId=${accountId}`
+      var parameters = defaultParams
+      getRequest({
+        extension: FinancialRepository.AccountCreditLimit.qry,
+        parameters: parameters
+      })
+        .then(res => {
+          if (res.list.length > 0){
+            const currencies = res.list.map(({ ...rest } , index) => ({
+                id : index,
+                ...rest
+            }))
+            formik.setValues({ currencies: currencies})
+
+          setStore(prevStore => ({
+            ...prevStore,
+            currencies: currencies,
+          }));
+          }
+        })
+        .catch(error => {
```

No empty catch blocks.